Changes

```diff
@@ -1,13 +1,28 @@
 Release history for Zonemaster component Zonemaster-LDNS
 
+
+3.0.0 2019-11-15
+
+ [API Change]
+ - Requires libldns 1.7.0 or 1.7.1 (#85)
+ - Requires OpenSSL 1.1.1 unless Ed25519 (algorithm 15)
+   has been disabled
+
+ [Changed]
```

Comment:
This is an addition, not a change. Please read https://semver.org/ and https://keepachangelog.com/en/1.0.0/.